zipline-loader-testing/src/engineMain/kotlin/app/cash/zipline/loader/testing/LoaderTestFixtures.kt

```diff
@@ -142,15 +142,15 @@ class LoaderTestFixtures(
       serializersModule = serializersModule,
       httpClient = httpClient,
       fetchers = listOf(
-        FsCachingFetcher(
```

Comment:
Why do we need to flip them again?

Reply:
For modules I wanna prefer fastest. Checksums ensure we have the right bytes so we want the most-local copy of the data we can find.

For manifests I wanna prefer freshest. We want server first, then cache, then embedded.

Reply:
(I think this makes the current API a hazard because there’s huge performance consequences of getting it wrong, but it’ll appear to work.)